spec/raven/integrations/rack_spec.rb

```diff
@@ -46,6 +46,22 @@
     stack.call(env)
   end
 
+  it 'should capture rails errors' do
```

Comment:
Can you just change this line to say 'should capture rails errors when ActionDispatch::ShowExceptions is enabled'?